src/DynamicMethodTrait.php

```diff
@@ -4,20 +4,147 @@
 
 trait DynamicMethodTrait {
 
-    function __call($fx, $args){
+    public function __call($method, $arguments)
+    {
+        if (($ret = $this->tryCall($method, $arguments))) {
```

Comment:
It's a tricky one. Basically some people may see this as mistake where `==` should be used instead of `=` , thats why (( )) are used. It's some sort of tradition from C days. I'll try to find reference.